.github/pull_request_template.md

```diff
@@ -0,0 +1,5 @@
+### Checklist
+- [ ] test changes in [status-desktop](https://github.com/status-im/status-desktop)
+- [ ] resolve breaking changes in [status-desktop](https://github.com/status-im/status-desktop) (renamed property, removed component, etc.)
+   - [ ] (pre-merge) adapt code to breaking changes
+   - [ ] (post-merge) bump StatusQ version
```

Comment:
Thanks! This is a good start. Here are a few more things we should include:

- [ ] Do commits follow [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)  ?
  - The scope should be the component's name e.g: `feat(StatusListItem): ... `
  - When adding new components, the scope is the module e.g: `feat(StatusQ.Controls): ...`
- [ ] Is this a breaking change?
  - If yes, make sure the commits use the dedicated `BREAKING CHANGE` section
- [ ] Add documentation if necessary (new component, new feature)
- [ ] Has sandbox been updated?
  - In case of new component, add new component page
  - In case of new features, add variation to existing component page
  - Nice to have: add it to the demo application as well